pkg/proc/eval.go

```diff
@@ -214,6 +221,93 @@ func (scope *EvalScope) EvalExpression(expr string, cfg LoadConfig) (*Variable,
 	return ev, nil
 }
 
+type scopeToEvalLookup struct {
```

Comment:
Why create the new type which simply embeds the other? Not wanting to pollute the API of `EvalScope`?

Reply:
Yes.